.gitignore

```diff
@@ -1,17 +1,16 @@
-*.pyc
-.DS_Store
-*.swp
+# Be sure to update .dockerignore to match .gitignore
 /venv/
 /static/
 /media/
-/docs/_build/
```

Comment:
This entry is not in the new updated ignore files, was that deliberate?

Reply:
I couldn't find any `docs` directory in the codebase, either on `main` or in history, so I assumed the rule is not useful.